File: DailyFlash Solutions/DailyFlash Solutions/62-DailyFlash_Solutions/01_April_SOlution_One/Python/Program0.py
from math import *

n = int(input("Enter the limit : "))
x,y = [ int(x) for x in input("Enter x and y : ").split()]

s=0.0

for i in range(1,n+1):
        s+=i*(x+y)/factorial(i)


print("Result of series is :",round(s,2))

File: DailyFlash Solutions/DailyFlash Solutions/62-DailyFlash_Solutions/01_April_SOlution_One/CPP/Program0.cpp
#include<stdio.h>

double pow(double,double);

int calculateFactorial(int num){

	int fact =1;
	while(num>0){
		fact *=num;
		num -= 1;
	}

	return fact;
}



int main(){

	int a,x,y;
	double res=0;

	printf("Enter the limit : ");
	scanf("%d",&a);
	printf("Enter X: ");
	scanf("%d",&x);
	printf("Enter Y: ");
	scanf("%d",&y);


	for(int i=1; i<=a; i++)
		res+=(double)(i)*(x+y)/(calculateFactorial(i));
		

	printf("Result of series is %.2lf\n",res);
	return 0;
}
